backend/src/main/kotlin/app/ehrenamtskarte/backend/auth/service/Authorizer.kt

```diff
@@ -139,12 +144,24 @@ object Authorizer {
         return user.projectId.value == projectId && user.role == Role.PROJECT_STORE_MANAGER.db_value
     }
 
-    fun mayAddApiTokensInProject(user: AdministratorEntity): Boolean =
-        user.role == Role.PROJECT_ADMIN.db_value
+    fun mayAddApiTokensInProject(user: AdministratorEntity): Boolean {
+        return transaction {
+            user.role == Role.PROJECT_ADMIN.db_value && ProjectEntity.find { Projects.project eq KOBLENZ_PASS_PROJECT }
```

Comment:
nit picky. I don't mind leaving as is, just wanted to share a thought:
I think it's better to check if token creation is allowed in a particular project by project configuration instead of project name (is selfServiceEnabled?)
And then this should be checked in mutation, because it won't be related to the rights of the particular user.

Reply:
You are right and i agree, it would be nicer, if we have this in a config file. Currently we only have this configs in administration and it would be nice in the long term to have a shared project config for all parts of the project. As my suggestion is consistent with how we did it in the backend everywhere else, i would keep it, but I agree with you we should think of a global config to avoid this.